tests/test_library.py

```
@@ -38,6 +39,12 @@ def test_toollib_getdict_type():
     assert ToolLibrarySchema._getdict_type() == "ToolLibrarySchema"
 
 
+def test_package_access():
+    lib = LibrarySchema()
+    assert lib.get("package", field="schema") is lib.package
+    assert isinstance(self.package, PackageSchema)
```

Ruff (0.13.1)

Undefined name `self`

(F821)